src/QMCWaveFunctions/tests/test_ConstantSPOSet.cpp

```diff
@@ -0,0 +1,193 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2023 QMCPACK developers.
+//
+// File developed by: Raymond Clay, rclay@sandia.gov, Sandia National Laboratories
+//
+// File created by: Raymond Clay, rclay@sandia.gov, Sandia National Laboratories
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#include "catch.hpp"
+#include "Configuration.h"
+#include "QMCWaveFunctions/OrbitalSetTraits.h"
+#include "QMCWaveFunctions/ConstantSPOSet.h"
+
+namespace qmcplusplus
+{
+//Ray:  Figure out how to template me on value type.
```

Comment:
For reference, this test uses TEMPLATE_TEST_CASE: https://github.com/QMCPACK/qmcpack/blob/19b99c950d1bb518332c69bda4adf823adab97d1/src/Numerics/tests/test_RotationMatrix3D.cpp#L20-L23

Reply:
Thank you!   I'll try to make more efficient use of template_test_case in the future.  For now, real vs. complex test cases are covered by letting the imaginary parts be zero.